tests/infrastructure/router.integration.coffee

```diff
@@ -0,0 +1,79 @@
+# =========== SETUP ============= #
+
+RouterTests = Space.namespace('RouterTests')
+
+Space.messaging.define Space.messaging.Command, 'RouterTests', {
+  Create: targetId: String
```

Comment:
Might be best to make this slightly more verbose for the purpose of clarity in the tests?

`CreateTestAggregate`